test/extended/util/test.go

```diff
@@ -91,7 +91,7 @@ func InitTest(dryRun bool) error {
 	// Ensure that Kube tests run privileged (like they do upstream)
 	TestContext.CreateTestingNS = createTestingNS
 
-	klog.V(2).Infof("Extended test version %s", version.Get().String())
+	klog.Infof("Extended test version %s", version.Get().String())
```

Comment:
This isn't the right place to print this.  It should only be printed in run suite, and we should print it with `fmt.Printf()`.

Reply:
Reworked to `openshift-tests run` only and sent to stderr.